lib/app/modules/uscore_v3.0.0/us_core_diagnosticreport_lab_sequence.rb

```diff
@@ -7,7 +7,7 @@ class USCore300DiagnosticreportLabSequence < SequenceBase
 
       description 'Verify that DiagnosticReport resources on the FHIR server follow the Argonaut Data Query Implementation Guide'
 
-      test_id_prefix 'DiagnosticReport' # change me
+      test_id_prefix 'USCDRLRR'
```

Comment:
Yeah, the name of the profile is "DiagnosticReport Profile for Laboratory Results Reporting".  I was considering changing the global prefix to "US" instead of "USC" and then maybe capping the regular length at 5 chars (unless we collide).  But I think this is good enough for now.

Reply:
I'm going to resolve this and if it is annoying enough in practice we can create a new ticket to clean it up more.